generated/python/gapic-google-cloud-iam-v1/README.rst

```diff
@@ -0,0 +1,78 @@
+GAPIC library for the Google Admin API
```

Comment:
Hmm... this should be hand-edited to say "Google IAM Admin API". We have a [Github issue](https://github.com/googleapis/toolkit/issues/270) to improve where we source this title (e.g. from https://github.com/googleapis/googleapis/pull/151/files#diff-b8750c321eaa4d100c91edbf078d08a3R7).